hera_qm/utils.py

```diff
@@ -138,7 +138,7 @@ def get_metrics_ArgumentParser(method_name):
     elif method_name == 'xrfi_run':
         a.prog = 'xrfi_run.py'
         a.add_argument('--infile_format', default='miriad', type=str,
-                       help='File format for input files. Default is miriad.')
+                       help='File format for input files. DEPRECATED, but kept for legacy.')
```

Comment:
Do we want to get fancy and actually have a `DeprecationWarning` displayed to the user if they supply this argument?

Reply:
Also, why is it deprecated? Because we use the generic `read` method in pyuvdata?